script.js

```diff
@@ -0,0 +1,139 @@
+ * 8.  Add className for divImg element.
+ * 9.  Append img element to divImg.
+ * 10. Add className for img element with help of template literal .
+ *     https: developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
+ * 11. Add src attribute for image.
+ *     https: developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/src
+ * 12. Append divTxt element to divRow.
+ * 13. Add className for divTxt.
+ * 14. Append text element to divTxt.
+ * 15. Append text node to text element.
+ *     https: developer.mozilla.org/en-US/docs/Web/API/Document/createTextNode
+ * 16. Append br element to body after all other elements have been added.
+ * 17  Run renderComponents function with ELEMENTS as a paramater.
+*/
+
+const ELEMENTS = [
+    {
+      name: "Sun",
+      src: "./img/SUN.jpg",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet.",
+    },
+    {
+      name: "Mercury",
+      src: "./img/MERCURY.jpg",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Venus",
+      src: "./img/VENUS.jpg",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Earth",
+      src: "./img/EARTH.jpg",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Mars",
+      src: "./img/MARS.jpg",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Jupiter",
+      src: "./img/JUPITER.jpg",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Saturn",
+      src: "./img/SATURN.png",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Uranus",
+      src: "./img/URAN.png",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Neptune",
+      src: "./img/NEPTUNE.png",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+  ]
+
+
+  const bodyElement = document.getElementById("Body");
+
+  // TODO: write a simillar function to render navigation elements.
+
+  const MENUARRAY = [
+    {
+      name: "Home",
+      adres: "home.html"
```

Comment:
```suggestion
      address "home.html"
```
"adres" is in Polish.

You can also consider changing the name of this attribute to `path`/`fileName` - but it's a minor thing at this point.

Also, those files don't exist, so those hyperlinks won't work at this point.

You could also do some exercise and hook some different behaviour of elements of the page when such links would be clicked (better then to transfer <a> elements to <buttons> and render some content when they are clicked). To do that you would have to read about event listeners in javascript.